modules/bootkube/resources/bootkube.sh

```diff
@@ -1,42 +1,55 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -e
 
-echo "Rendering Kubernetes core manifests..."
-
-# shellcheck disable=SC2154
-/usr/bin/podman run \
-    --volume "$PWD:/assets:z" \
-    --volume /etc/kubernetes:/etc/kubernetes:z \
-    "${kube_core_renderer_image}" \
-    --config=/assets/kco-config.yaml \
-    --output=/assets
-
-echo "Rendering MCO manifests..."
-
-# shellcheck disable=SC2154
-/usr/bin/podman run \
-    --user 0 \
-    --volume "$PWD:/assets:z" \
-    "${machine_config_operator_image}" \
-    bootstrap \
-    --etcd-ca=/assets/tls/etcd-client-ca.crt \
-    --root-ca=/assets/tls/root-ca.crt \
-    --config-file=/assets/manifests/cluster-config.yaml \
-    --dest-dir=/assets/mco-bootstrap \
-    --images-json-configmap=/assets/manifests/machine-config-operator-01-images-configmap.yaml
-
-mkdir -p /etc/kubernetes/manifests/
-mkdir -p /etc/mcc/bootstrap/
-mkdir -p /etc/ssl/mcs/
-# Bootstrap MachineConfigController uses /etc/mcc/bootstrap/manifests/ dir to
-# 1. read the controller config rendered by MachineConfigOperator
-# 2. read the default MachineConfigPools rendered by MachineConfigOperator
-# 3. read any additional MachineConfigs that are needed for the default MachineConfigPools.
-cp -r "$PWD/mco-bootstrap/manifests" /etc/mcc/bootstrap/manifests
-# /etc/ssl/mcs/tls.{crt, key} are locations for MachineConfigServer's tls assets.
-cp "$PWD/tls/machine-config-server.crt" /etc/ssl/mcs/tls.crt
-cp "$PWD/tls/machine-config-server.key" /etc/ssl/mcs/tls.key
-cp "$PWD/mco-bootstrap/machineconfigoperator-bootstrap-pod.yaml" /etc/kubernetes/manifests/
+mkdir --parents /etc/kubernetes/manifests/
+
+if [ ! -d "$PWD/kco-bootstrap" ]
+then
+	echo "Rendering Kubernetes core manifests..."
+
+	# shellcheck disable=SC2154
+	podman run \
+		--volume "$PWD:/assets:z" \
+		--volume /etc/kubernetes:/etc/kubernetes:z \
+		"${kube_core_renderer_image}" \
+		--config=/assets/kco-config.yaml \
+		--output=/assets/kco-bootstrap
+
+	cp --recursive "$PWD/kco-bootstrap/bootstrap-configs" /etc/kubernetes/bootstrap-configs
+	cp --recursive "$PWD/kco-bootstrap/bootstrap-configs" .
```

Comment:
```console
	cp --recursive "$PWD/kco-bootstrap/bootstrap-configs" /etc/kubernetes/bootstrap-configs
	cp --recursive "$PWD/kco-bootstrap/bootstrap-configs" .
```
the end goal is to end up at `/etc/kubernetes/bootstrap-configs`; 
why do both? 
or atleast copy to $PWD and then copy $PWD one to `/etc/kubernetes/bootstrap-configs`

Reply:
The second cp was to satisfy the conditional above. I thought those files in /etc were removed by kubelet. If not, we could just read /etc in the conditional.

Reply:
No the /etc/kubernetes/bootstrap-configs is not deleted by kubelet.

It is read by `bootstrap-control-plane`

Reply:
We can 
* Change conditional to /etc and drop cp .
* Keep it as is, no big deal

Reply:
Keeping it as is, seems okay for now. 👌